app/assets/javascripts/components/reviews/fullreview.js.jsx

```diff
@@ -0,0 +1,59 @@
+var FullReview = React.createClass({
+  upVote: function(e){
+    e.preventDefault();
+    this.updateVotes({voteValue: 1, reviewId: this.props.data.id})
```

Comment:
It would be better if the `updateVote` took care of the increment/decrement rather than sending a `voteValue`.